packages/camera/camera_platform_interface/lib/src/types/file_format.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The format in which images should be returned from the camera.
+enum FileFormat {
```

Comment:
This seems redundant to `ImageFormatGroup`: https://github.com/flutter/packages/blob/main/packages/camera/camera_platform_interface/lib/src/types/image_format_group.dart.

Should `heif` just bee added to this?

Reply:
Correct me if I'm wrong, but these are for pixel definitions, right?

Reply:
Yea, I think you're right. The issue is that we already have jpeg in the other one and it seems we are adding it in two places. But I do agree we should probably make this distinction.

I would change the name of this to `ImageFileFormat`.